### PR TITLE
Balance check done before sending messages

### PR DESCRIPTION
**Pull request by jeandemeusy**
In this PR, a channel balance check is done before sending messages. If the balance is too low, only a subset of messages worth the channel balance are sent.

### CODE REVIEW

File: ct-app/postman/postman_tasks.py [hunk 1]
```diff
@@ -31,6 +31,23 @@ class TaskStatus(Enum):
...
+    :return: Channel balance of the address.
+    """
+    channels = await api.outgoing_channels(address)
+
+    channel = list(filter(lambda c: c.peerAddress == address, channels))
+
+    if len(channel) == 0:
+        return 0
+    else:
+        return int(channel[0].balance) / 1e18
```
Comment (Teebor-Choka) [inline ct-app/postman/postman_tasks.py:48]: You could use `reduce` to make this a one liner: https://www.geeksforgeeks.org/reduce-in-python/

File: ct-app/tests/test_postman.py [hunk 2]
```diff
@@ -47,6 +47,7 @@ async def test_async_send_1_hop_message_hit_timeout():
         expected_count=10,
         node_list=["node1", "node2", "node3"],
         node_index=0,
+        ticket_price=0.001,
```
Comment (Teebor-Choka) [inline ct-app/tests/test_postman.py:50]: Should this be like a global config?

File: ct-app/postman/postman_tasks.py (hunk 1 above)
Comment (jeandemeusy) [reply ct-app/postman/postman_tasks.py:48]: Operation is quite simple, the perf improvement from using functools is super small. Could be done for the sake of code beauty actually. Will keep this in mind for my refactoring that is still ongoing

File: ct-app/tests/test_postman.py (hunk 2 above)
Comment (jeandemeusy) [reply ct-app/tests/test_postman.py:50]: Could be.

**Review by Teebor-Choka — APPROVED** [review]

**Pull request merged**